src/ChatColor.h

```diff
@@ -32,7 +32,7 @@ class cChatColor
 	static const char * White;
 
 	// Styles
-	// source: http://wiki.vg/Chat
+	// source: https://minecraft.wiki/w/Text_format
```

Comment:
don't link to a disambiguation page
```suggestion
	// source: https://minecraft.wiki/w/Formatting_codes
```